src/ziggurat/producer.clj

```diff
@@ -58,7 +59,7 @@
   (reduce (fn [producer-map [stream-config-key stream-config]]
             (let [producer-config (:producer stream-config)]
               (if (some? producer-config)
-                (assoc producer-map stream-config-key (build-producer-config-properties producer-config))
+                (assoc producer-map stream-config-key (ssl-properties/build-ssl-properties (build-producer-config-properties producer-config)))
```

Comment:
we should add ssl configs in config.clj where we're adding properties for stream, consumer and producer APIs.

[Same comment for streams.clj as well]